src/App.css

```diff
@@ -0,0 +1,57 @@
+.App {
```

Comment:
Do not use capitals for CSS class names, I know the template comes with it lol, rename 'em